flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java

```diff
@@ -51,11 +61,28 @@ public void actionPerformed(AnActionEvent e) {
         }
       }
     }
-    NewProjectUtil.createNewProject(getEventProject(e), wizard);
+    createNewProject(getEventProject(e), wizard);
   }
 
   private JBList fetchPrivateField(@SuppressWarnings("SameParameterValue") String fieldName) {
     return ReflectionUtil // Fetching a private field.
       .getField(ProjectTypeStep.class, myProjectTypeStep, JBList.class, fieldName);
   }
+
+  private static void createNewProject(Project projectToClose, AbstractProjectWizard wizard) {
+    final boolean proceed = ProgressManager.getInstance().runProcessWithProgressSynchronously(() -> {
+      ProjectManager.getInstance().getDefaultProject(); //warm up components
+    }, ProjectBundle.message("project.new.wizard.progress.title"), true, null);
+    if (!proceed) return;
+    if (!wizard.showAndGet()) {
+      return;
+    }
+
+    final Project newProject = NewProjectUtil.createFromWizard(wizard, projectToClose);
```

Comment:
A comment here saying that we're the `project` view (as opposed to the default `android` view) in order for Dart resources to be visible at project creation?

Reply:
Good idea, done. Also, reformat, reorganize, optimize imports. But no code changes so I'll just merge it.